packages/devtools_app/lib/src/ui/icons.dart

```diff
@@ -285,6 +291,111 @@ class ThemedImageIcon extends StatelessWidget {
   }
 }
 
+class WidgetIcons {
+  static String getAssetName(String widgetType) {
+    if (widgetType == null) {
+      return null;
+    }
+
+    return _iconMap[_stripBrackets(widgetType)];
+  }
+
+  static String _stripBrackets(String widgetType) {
+    final bracketIndex = widgetType.indexOf('<');
```

Comment:
can you add a comment describing what the expected input looks like?

Reply:
Yea I will